packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts

```diff
@@ -260,7 +260,7 @@ export function generateGatsbyImageFieldConfig(
   IGatsbyImageDataArgs
 > {
   return {
-    type: `JSON`,
+    type: `GatsbyImageData!`,
```

Comment:
This is a breaking change in sense that it requires users to bump `gatsby` package (to version that actually creates that scalar) (this comment apply to all places that make similar swap)